services/kube-eleven/Dockerfile

```diff
@@ -39,7 +39,7 @@ WORKDIR /go/services/kube-eleven/server
 #Compile the golang code, CGO_ENABLE=0 removes cross compile dependencies
 RUN CGO_ENABLED=0 go build
 
-FROM alpine
+FROM docker.io/library/alpine
```

Comment:
Can we pin the version of the `alpine` image to some reasonable version while we are at it?

Reply:
absolutely, we could do `3.17` or `3.17.1`, depending on for how long we want to call it a day (minor version tag is more precise, although it still isn't a hash).